Makefile

```diff
@@ -48,14 +69,13 @@ mod-download:
 .PHONY: update-deps
 update-deps:
 	mkdir -p $(DEPSGOBIN)
-	GOBIN=$(DEPSGOBIN) go install github.com/solo-io/protoc-gen-ext
-	GOBIN=$(DEPSGOBIN) go install github.com/solo-io/protoc-gen-openapi
-	GOBIN=$(DEPSGOBIN) go install golang.org/x/tools/cmd/goimports
-	GOBIN=$(DEPSGOBIN) go install github.com/golang/protobuf/protoc-gen-go
-	GOBIN=$(DEPSGOBIN) go install github.com/envoyproxy/protoc-gen-validate
-	GOBIN=$(DEPSGOBIN) go install github.com/golang/mock/gomock
-	GOBIN=$(DEPSGOBIN) go install github.com/golang/mock/mockgen
-	GOBIN=$(DEPSGOBIN) go install github.com/onsi/ginkgo/ginkgo
```

Comment:
explicitly calling out that this is no longer present (intentional, I'm sure).  Specifically, the +/- delta made it a *bit* hard to immediately see

Reply:
Yeah I always wondered if all of those installs are necessary. I didn't explore the others, but I moved just the installs required for tests (ie installing a particular version of Ginkgo) into its own target.